src/server/_limiter.py

```diff
@@ -71,12 +71,13 @@ def get_multiples_count(request):
 def check_signals_allowlist(request):
     signals_allowlist = {":".join(ss_pair) for ss_pair in DashboardSignals().srcsig_list()}
     request_signals = []
-    if "signal" in request.args.keys():
-        request_signals += extract_strings("signal")
-    if "signals" in request.args.keys():
-        request_signals += extract_strings("signals")
-    if "data_source" in request.args:
-        request_signals = [f"{request.args['data_source']}:{request_signal}" for request_signal in request_signals]
+    request_args = request.args.keys()
+    if "signal" in request_args or "signals" in request_args:
+        source_signal_sets = parse_source_signal_sets()
+        for source_signal in source_signal_sets:
+            if isinstance(source_signal.signal, list):
+                for signal in source_signal.signal:
+                    request_signals.append(f"{source_signal.source}:{signal}")
```

Comment:
we dont want to let users specify the wildcard "*" that matches all signals:
```suggestion
            if isinstance(source_signal.signal, bool):
                # wildcards are not allowed
                return False
            for signal in source_signal.signal:
                request_signals.append(f"{source_signal.source}:{signal}")
```

Reply:
sorry, that wasnt specific enough:  we want to make sure users are rate limited when they specify the wildcard "*" that matches all signals